raiden/transfer/mediated_transfer/target.py

```diff
@@ -240,10 +240,7 @@ def handle_block(target_state, channel_state, block_number):
         events = [failed]
 
     elif target_state.state != 'waiting_close':  # only emit the close event once
-        # TODO: to be removed
-        events = events_for_close(target_state, channel_state, block_number)
```

Comment:
the same thing must be done for the mediator

Reply:
I removed it as well, all tests pass.